operations/agent-flow-mixin/dashboards/cluster-node.libsonnet

```diff
@@ -0,0 +1,257 @@
+        ])
+      ),
+      // Gossip ops/sec
+      (
+        panel.new('Gossip ops/s', 'timeseries') +
+        panel.withDescription(|||
+          Gossip ops/sec for this node.
+        |||) +
+        panel.withPosition({ x: 12, y: 1, w: 12, h: 8 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='rate(cluster_node_gossip_received_events_total{instance="$instance"}[$__rate_interval])',
+            legendFormat='{{event}}'
+          ),
+        ])
+      ),
+      // Known peers
+      (
+        panel.new('Known peers', 'stat') +
+        panel.withDescription(|||
+          Known peers to the node (including the local node).
+        |||) +
+        panel.withPosition({ x: 0, y: 9, w: 12, h: 8 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='sum(cluster_node_peers{instance="$instance"})',
+          ),
+        ]) +
+        panel.withUnit('suffix:peers')
+      ),
+      // Peers by state
+      (
+        panel.new('Peers by state', 'timeseries') +
+        panel.withDescription(|||
+          Known peers to the node by state.
+        |||) +
+        panel.withPosition({ x: 12, y: 9, w: 12, h: 8 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='cluster_node_peers{instance="$instance"}',
+            legendFormat='{{state}}',
+          ),
+        ]) +
+        panel.withUnit('suffix:nodes')
+      ),
+      // Gossip Transport row
+      (
+        panel.new('Gossip Transport', 'row') +
+        panel.withPosition({ h: 1, w: 24, x: 0, y: 17 })
+      ),
+      // Transport bandwidth
+      (
+        panel.new('Transport bandwidth', 'timeseries') +
+        panel.withPosition({
+          h: 8,
+          w: 8,
+          x: 0,
+          y: 18,
+        }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='rate(cluster_transport_rx_bytes_total{instance="$instance"}[$__rate_interval])'
+          ),
+          panel.newQuery(
+            expr='-1 * rate(cluster_transport_tx_bytes_total{instance="$instance"}[$__rate_interval])'
+          ),
+        ]) +
+        panel.withUnit('Bps')
+      ),
+      // Packet write success rate
+      (
+        panel.new('Packet write success rate', 'timeseries') +
+        panel.withPosition({
+          h: 8,
+          w: 8,
+          x: 8,
+          y: 18,
+        }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='1 - ( \r\n    rate(cluster_transport_tx_packets_failed_total{instance="$instance"}[$__rate_interval]) / \r\n    rate(cluster_transport_tx_packets_total{instance="$instance"}[$__rate_interval])\r\n)',
+            legendFormat='Tx success %',
+          ),
+          panel.newQuery(
+            expr='1 - ( \r\n    rate(cluster_transport_rx_packets_failed_total{instance="$instance"}[$__rate_interval]) / \r\n    rate(cluster_transport_rx_packets_total{instance="$instance"}[$__rate_interval])\r\n)',
+            legendFormat='Rx success %',
```

Comment:
Can we make these multiline Jsonnet strings just so it's easier for us to read? Ditto with any other query with \r\n in it.